<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrafficConfigurationDialog</class>
 <widget class="QDialog" name="TrafficConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Traffic Setup</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,0">
     <item>
      <widget class="PersonalGraphicsView" name="graphicsView">
       <property name="renderHints">
        <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
       </property>
       <property name="dragMode">
        <enum>QGraphicsView::RubberBandDrag</enum>
       </property>
       <property name="transformationAnchor">
        <enum>QGraphicsView::AnchorUnderMouse</enum>
       </property>
       <property name="viewportUpdateMode">
        <enum>QGraphicsView::FullViewportUpdate</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item alignment="Qt::AlignHCenter">
        <widget class="QToolButton" name="zoomIn">
         <property name="icon">
          <iconset resource="../resources/Icons.qrc">
           <normaloff>:/icons/icons/zoomin.png</normaloff>:/icons/icons/zoomin.png</iconset>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QToolButton" name="zoomOut">
         <property name="icon">
          <iconset resource="../resources/Icons.qrc">
           <normaloff>:/icons/icons/zoomout.png</normaloff>:/icons/icons/zoomout.png</iconset>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QToolButton" name="zoomRestore">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/Icons.qrc">
           <normaloff>:/icons/icons/zoomrestore.png</normaloff>:/icons/icons/zoomrestore.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="zoomScale">
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.150000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1">
       <item>
        <widget class="QListWidget" name="listConf">
         <property name="editTriggers">
          <set>QAbstractItemView::CurrentChanged|QAbstractItemView::SelectedClicked</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>340</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Traffic Setup</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Source selected:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelNodeSelected">
              <property name="text">
               <string>None</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>90</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="buttonAddTraffic">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Add Traffic</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listTraffic">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialogButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PersonalGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>View/PersonalGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
